frontend/packages/integration-tests-cypress/tests/storage/clone.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { testName, checkErrors } from '../../support';
+import { listPage } from '../../views/list-page';
+import { PVC, testerDeployment } from '../../mocks/snapshot';
+import { detailsPage, DetailsPageSelector } from '../../views/details-page';
+import { modal } from '../../views/modal';
+
+import { resourceStatusShouldContain } from '../../views/common';
+
+const cloneName = `${PVC.metadata.name}-clone`;
+const cloneSize = '2';
+
+describe('Clone Tests', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+    cy.exec(`echo '${JSON.stringify(PVC)}' | oc apply -n ${testName} -f -`);
+    cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc apply -n ${testName} -f -`);
+    cy.clickNavLink(['Storage', 'Persistent Volume Claims']);
```

Comment:
Just a heads up that when #6521 merges, this should be changed to `nav.sidenav.clickNavLink(..`.   A bunch of new helper methods added with `nav.sidenav`.

Reply:
ack!

Reply:
Done